core/src/tag_utils.rs

```diff
@@ -92,30 +104,38 @@ impl SwfMovie {
     /// `LoaderInfo.bytesTotal` is set to the actual value, but no data is available,
     /// and `LoaderInfo.parameters` is empty.
     pub fn fake_with_compressed_len(swf_version: u8, compressed_len: usize) -> Self {
+        let url = "file:///".to_string();
```

Comment:
Pretty sure this is going to be a problem in the future. We need something like #17717.

Reply:
That's right, added TODOs for now, but this will need a lot more tests for sure